tools/cross/format.py

```diff
@@ -114,19 +134,39 @@ def git_get_modified_files(
         return files_in_diff
 
 
+def format_cpp(files_in_diff: Optional[Set[str]], check: bool) -> None:
+    cpp_files = set(find_files_by_exts("src/workerd", (".c++", ".h")))
+
+    if files_in_diff:
+        cpp_files &= files_in_diff
+
+    clang_format(list(cpp_files), check)
+
+
+def format_js(files_in_diff: Optional[Set[str]], check: bool) -> None:
+    js_files = set(find_files_by_exts("src", (".js", ".ts", ".json")))
```

Comment:
nit: I'd separate json to another function, we can still use prettier but does it really fit with js and ts in this list?